monkestation/_maps/random_rooms.dm

```diff
@@ -0,0 +1,85 @@
+/datum/map_template/random_room
```

Comment:
This section needs to go.
All of this is already defined in the main file itself, doesn't need a second, identical, definition